openwisp_controller/connection/tests/test_transaction_blocks.py

```diff
@@ -0,0 +1,54 @@
+from unittest import mock
+
+from django.test import TransactionTestCase
+
+from .. import settings as app_settings
+from .utils import CreateConnectionsMixin
+
+
+class TestTransactionBlocks(CreateConnectionsMixin, TransactionTestCase):
```

Comment:
let's rename it to `TestTransactions`, please also add this test class to the sample connection app